Add more information about Discovery Cluster

candleindark opened the pull request:
Add information for setting up VPN and ProxyJump to access Discovery and some pointers to learn more about Discovery.

candleindark committed:
Add more information about Discovery Cluster

Add information for setting up VPN and ProxyJump to access Discovery
and some pointers to learn more about Discovery.

yarikoptic commented on conboarding.md:
```diff
@@ -39,7 +39,32 @@ How much does it cost to run things?
+                ProxyJump smaug.dartmouth.edu
+                ServerAliveInterval 30
+                GSSAPIAuthentication yes
+                GSSAPIDelegateCredentials yes
```
GSSAPI setting is useful only if you use `kinit` as https://dbic-handbook.readthedocs.io/en/latest/discovery.html#password-less-authentication-for-ssh describes, add more information or point to that page at least.

candleindark replied:
The GSSAPI settings is now taken out. The handbook from the imaging center is referenced later in the document.

candleindark committed:
Take out the GSSAPI settings in example SSH config

The GSSAPI is only relevant in setting up with kinit
and is not needed for ProxyJump

asmacdo reviewed (approved):
lgtm

yarikoptic commented on conboarding.md:
```diff
@@ -39,7 +39,30 @@ How much does it cost to run things?
+                user <your username>
+         
+            Host discovery discovery.dartmouth.edu
+                HostName discovery.dartmouth.edu
```
To avoid ssh trying to use ssh keys and overall fail before trying password, should have following:

```suggestion
                HostName discovery.dartmouth.edu
                PreferredAuthentications gssapi-with-mic,hostbased,keyboard-interactive,password
```

asmacdo committed:
Update conboarding.md

Co-authored-by: Yaroslav Halchenko <debian@onerussian.com>
